test/Microsoft.AspNetCore.Hosting.Tests/WebHostBuilderTests.cs

```diff
@@ -149,6 +149,27 @@ public void UseCaptureStartupErrorsHonored()
         }
 
         [Fact]
+        public void ConfigureServices_CanBeCalledMultipleTimes()
+        {
+            var hostBuilder = new WebHostBuilder()
+                .UseServer(new TestServer())
+                .ConfigureServices(services =>
+                {
+                    services.AddTransient<ServiceA, ServiceA>();
+                })
+                .ConfigureServices(services =>
+                {
+                    services.AddTransient<ServiceB, ServiceB>();
+                })
+                .Configure(app => { });
+
+            var host = hostBuilder.Build();
+
+            Assert.NotNull(host.Services.GetRequiredService<ServiceA>());
+            Assert.NotNull(host.Services.GetRequiredService<ServiceB>());
+        }
+
+        [Fact]
```

Comment:
Should we add a test to check for the order that the ConfigureServices are called?